src/utils/authentication/authentication.service.ts

```diff
@@ -0,0 +1,64 @@
+import {
```

Comment:
This file will probably change quite a bit once the backend is ready. Consider it a draft for now.